packages/react/src/components/Popover/next/Popover.stories.js

```diff
@@ -127,3 +127,61 @@ Playground.story = {
     (story) => <div className="mt-10 flex justify-center">{story()}</div>,
   ],
 };
+
+export const Automate = () => {
```

Comment:
```suggestion
export const AutoAlign = () => {
```

Would this kind of story title make sense? `Automate` wasn't immediately clear to me but let me know if you feel differently!

Reply:
Agreed! I like AutoAlign as a title much better